packages/front-end/services/features.ts

```diff
@@ -204,9 +205,12 @@ export function getVariationColor(i: number) {
   return colors[i % colors.length];
 }
 
-export function useAttributeSchema() {
-  const { attributeSchema } = useOrgSettings();
-  return attributeSchema || [];
+export function useAttributeSchema(showArchived = false) {
+  let attributeSchema = useOrgSettings().attributeSchema || [];
+  if (!showArchived) {
+    attributeSchema = attributeSchema.filter((s) => !s.archived);
+  }
```

Comment:
We might want to put this in a `useMemo` hook.  If `attributeSchema` is used as a dependency anywhere, as it currently stands it will cause it to re-render every time.  Ideally, it should only re-render if `useOrgSettings().attributeSchema` or `showArchived` changes.